Respect forms with enctype set for view transitions (#9466)
withastro/astro: .changeset/rude-geckos-rush.md

---
'astro': patch
---

> Respect forms with enctype set for view transitions

Reviewer: ```suggestion
Updates view transitions behavior with forms to respect `enctype`
```
